docs/ui/components/Home/sections/JoinTheCommunity.tsx

```diff
@@ -29,44 +32,57 @@ export function JoinTheCommunity() {
           title="Discord and Forums"
           description="Join our Discord to chat or ask questions."
           link="https://chat.expo.dev"
-          icon={<DiscordIcon className="icon-lg text-palette-white" />}
+          icon={<DiscordIcon className="icon-xl text-palette-white" />}
           iconClassName="bg-[#3131E8]"
           shouldLeakReferrer
         />
         <CommunityGridCell
           title="GitHub"
-          description="View our SDK, submit a PR, or report an issue."
+          description="View SDK and docs code, submit a PR, or report an issue."
           link="https://github.com/expo/expo"
-          iconClassName="bg-palette-gray10"
-          icon={<GithubIcon className="icon-lg text-palette-white" />}
+          iconClassName="bg-palette-gray11 dark:bg-palette-gray7"
+          icon={<GithubIcon className="icon-xl text-palette-white" />}
         />
         <CommunityGridCell
-          title="X"
-          description="Follow Expo on X for news and updates."
-          link="https://x.com/expo"
-          icon={<XLogoIcon className="icon-lg text-palette-white" />}
-          iconClassName="bg-[#000000]"
+          title="YouTube"
+          description="Visit our chnnel, explore guides and other content."
```

Comment:
```suggestion
          description="Follow our channel to explore tutorials and other content."
```

Fix the typo, and let's update the description.